src/parse.c

```diff
@@ -1423,6 +1437,7 @@ static const mapping_entry_handler routes_handlers[] = {
     {"type", YAML_SCALAR_NODE, handle_routes_type},
     {"via", YAML_SCALAR_NODE, handle_routes_ip, NULL, route_offset(via)},
     {"metric", YAML_SCALAR_NODE, handle_routes_metric},
+    {"mtu", YAML_SCALAR_NODE, handle_routes_mtu},
```

Comment:
```suggestion
    {"mtu", YAML_SCALAR_NODE, handle_routes_guint, NULL, route_offset(mtubytes)},
```
We should be making use of a more generic `handle_routes_guint` function here, to avoid duplication.
Actually, we could/should use a more generic approach for `handle_routes_metric` and `handle_routes_table` as well...